lib/meson.build

```diff
@@ -0,0 +1,46 @@
+
+lib_include_dir = [include_directories('.')]
+
+libvfio_user_cflags = cc.get_supported_arguments([
+  '-fvisibility=hidden',
+])
```

Comment:
Please consider passing the `gnu_symbol_visibility: 'hidden'` kwarg to the library instead. It's available since Meson 0.48.0.

Reply:
Yep, that works and is more portable